semanticcpg/src/test/scala/io/shiftleft/semanticcpg/language/types/structure/StructureTests.scala

```diff
@@ -25,7 +25,7 @@ class StructureTests extends WordSpec with Matchers {
     }
 
     "should allow traversing to type declarations" in {
-      cpg.file.typeDecl.name.toSet shouldBe Set("foo")
+      cpg.file.typeDecl.name.toSet shouldBe Set("foo", "void", "int")
```

Comment:
The other two types are now available through this traversal due to the empty file `""`.